internal/rpchelp/helpdescs_en_US.go

```diff
@@ -0,0 +1,399 @@
+	"gettransactiondetailsresult-amount":   "The amount of a received output",
+	"gettransactiondetailsresult-fee":      "The included fee for a sent transaction",
+
+	// ImportPrivKeyCmd help.
+	"importprivkey--synopsis": "Imports a WIF-encoded private key to the 'imported' account.",
+	"importprivkey-privkey":   "The WIF-encoded private key",
+	"importprivkey-label":     "Unused (all imported addresses belong to the imported account)",
+	"importprivkey-rescan":    "Rescan the blockchain (since the genesis block) for outputs controlled by the imported key",
+
+	// KeypoolRefillCmd help.
+	"keypoolrefill--synopsis": "DEPRECATED -- This request does nothing since no keypool is maintained.",
+	"keypoolrefill-newsize":   "Unused",
+
+	// ListAccountsCmd help.
+	"listaccounts--synopsis":       "Returns a JSON object of all accounts and their balances.",
+	"listaccounts-minconf":         "Minimum number of block confirmations required before an unspent output's value is included in the balance",
+	"listaccounts--result0--desc":  "JSON object with account names as keys and bitcoin amounts as values",
+	"listaccounts--result0--key":   "The account name",
+	"listaccounts--result0--value": "The account balance valued in bitcoin",
+
+	// ListLockUnspentCmd help.
+	"listlockunspent--synopsis": "Returns a JSON array of outpoints marked as locked (with lockunspent) for this wallet session.",
+
+	// TransactionInput help.
+	"transactioninput-txid": "The transaction hash of the referenced output",
+	"transactioninput-vout": "The output index of the referenced output",
+
+	// ListReceivedByAccountCmd help.
+	"listreceivedbyaccount--synopsis":        "Returns a JSON array of objects listing all accounts and the total amount received by each account.",
+	"listreceivedbyaccount-minconf":          "Minimum number of block confirmations required before a transaction is considered",
+	"listreceivedbyaccount-includeempty":     "Unused",
+	"listreceivedbyaccount-includewatchonly": "Unused",
+
+	// ListReceivedByAccountResult help.
+	"listreceivedbyaccountresult-account":       "The name of the account",
+	"listreceivedbyaccountresult-amount":        "Total amount received by payment addresses of the account valued in bitcoin",
+	"listreceivedbyaccountresult-confirmations": "Number of block confirmations of the most recent transaction relevant to the account",
+
+	// ListReceivedByAddressCmd help.
+	"listreceivedbyaddress--synopsis":        "Returns a JSON array of objects listing wallet payment addresses and their total received amounts.",
+	"listreceivedbyaddress-minconf":          "Minimum number of block confirmations required before a transaction is considered",
+	"listreceivedbyaddress-includeempty":     "Unused",
+	"listreceivedbyaddress-includewatchonly": "Unused",
+
+	// ListReceivedByAddressResult help.
+	"listreceivedbyaddressresult-account":           "Unset",
```

Comment:
I've intentionally avoided setting that field to anything meaningful since Bitcoin Core has deprecated it.